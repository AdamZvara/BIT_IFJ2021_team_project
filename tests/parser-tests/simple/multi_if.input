require "ifj21"


function foo()
	local a : string = "Ahoj\n"
	local b : integer = 5
	b = 5 * 2
	write(b,"\n")
	write(a)
	if b >= 10 then
		
	else
		write("sem by ses nemel dostat\n")
	end
	if b <= 10 then
		local b : integer = 1
		write("toto je jine b",b,"\n")
	else end
end

foo()
write("konec programu\n")

