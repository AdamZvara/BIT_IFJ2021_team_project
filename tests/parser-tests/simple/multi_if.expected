10
Ahoj
toto je jine b1
konec programu
